lib/pavilion/plugins/results/filecheck.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+from pavilion import config
+from pavilion import result_parsers
+from pavilion import utils
+import errno
+import glob
+import os
+import sys
+import yaml_config as yc
+
+
+class Filecheck(result_parsers.ResultParser):
+    """Checks the working directory for a given file.
+    The parser will tell the user if the filename exists or not. """
+
+    def __init__(self):
+        super().__init__(name='filecheck',
+                         description="Checks working directory"
+                         "for a given file")
+
+    def get_config_items(self):
+        # Result parser consists of 1 string elem: filename.
+        config_items = super().get_config_items()
+        config_items.extend([
```

Comment:
You don't need this argument. There is already a generic 'files' argument that the result parser uses to find (and open for you) specific files. See https://github.com/hpc/pavilion2/blob/6545b10d5f084ba0f8c996000f9896e39361c74b/lib/pavilion/result_parsers.py#L193

The files provide via the 'files' argument are found and opened for you by default, but you can change that behavior for your result parser plugin with arguments to __init__ (https://pavilion2.readthedocs.io/en/latest/source/result_parsers.html#pavilion.result_parsers.ResultParser.__init__)